design/design-doc-observe-only-resources.md

```diff
@@ -83,6 +83,10 @@ updated with the latest observation of the resource.
 
 ### Management Policy
 
+** Important Note ** The management policy was significantly changed in a
```

Comment:
Any way to get this rendered more eye-catching?

<img width="1042" alt="Screenshot 2023-08-01 at 17 40 21" src="https://github.com/crossplane/crossplane/assets/9900707/344cf580-8130-4000-964a-43fd58cbd490">

Reply:
Ill try to find a way

Reply:
Thanks @phisco, was looking for something like that!

Reply:
![Screenshot from 2023-08-01 17-00-56](https://github.com/crossplane/crossplane/assets/46844730/a1c4a44b-ffa2-4ad8-93fc-7b7cf5cf715c)